### PR TITLE
[PLAYER-167] change countdown time from string to number

### PR DESCRIPTION
**Pull request by ksunika**

### CODE REVIEW

**Comment by ksunika** [comment]
@mlen, @rcvrgs, @akshaydch11, is this change ok with you?

**Comment by rcvrgs** [comment]
I'm looking into this one. Verifying if we need to change code.

File: skin-schema.json [hunk 1]
```diff
@@ -474,8 +473,7 @@
         },
         "timeToShow": {
           "description": "Optional floating-point parameter used to set the time relative to the video length to show the Up Next screen. When timeToShow is set to a value greater than 1.0, the player interprets that number as the number of seconds before the end of the video to show the Up Next screen. However, when timeToShow is less than 1.0, the player treats the decimal value as a percentage of the video length and shows the Up Next screen at that point in the video. For example, if timeToShow is set to 0.9 in a 60-second video, the Up Next screen will show at 54 seconds. If timeToShow is set to 5.0 in a 60-second video, the Up Next screen will show at 55 seconds.",
-          "type": "string",
-          "pattern": "^[0-9]+$|^[0-9]{1,2}%$",
```
Comment (rcvrgs) [inline skin-schema.json]: Just catch this. timeToShow is supposed to accept percentage, which I don't think we've tested but we'd lose this functionality if we change it to the number pattern only. Are you ok with it? We can talk offline about it.

Comment (rcvrgs) [inline skin-schema.json]: Just to confirm, I just saw that in our side we do accept the percentage, so we would be losing this.

**Comment by rcvrgs** [comment]
We talked offline and after @ksunika's explanation we understand this better. We're merging this! Although we need to update some code in PBA to fully support these changes.

**Pull request merged**